boards/arm/thingy53_nrf5340/thingy53_nrf5340_fem.dts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/ {
+	nrf_radio_fem: nrf21540_fem {
+		compatible = "nordic,nrf21540-fem";
+		rx-en-gpios = <&gpio1 11 GPIO_ACTIVE_HIGH>;
+		mode-gpios  = <&gpio1 12 GPIO_ACTIVE_HIGH>;
+		pdn-gpios   = <&gpio1 10 GPIO_ACTIVE_HIGH>;
+		tx-en-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
+		spi-if = <&nrf_radio_fem_spi>;
+	};
+};
+
+fem_spi: &spi3 {
```

Comment:
Just realized we cannot do it like that. This file (`*_fem.dts`) is included form net core too, but net core does not have `spi3` at all.

Reply:
We need to find another way to do it, im taking a look into this.